src/Application.vala

```diff
@@ -17,7 +17,8 @@
 * Boston, MA 02110-1301 USA
 */
 
-public class PantheonTerminal.TerminalApp : Gtk.Application {
+namespace PantheonTerminal {
+public class TerminalApp : Gtk.Application {
```

Comment:
Why did you move the Namespace out of the class name? I think we are using the previous method in the rest of the apps :)

Reply:
Not sure - I guess there was a reason at the time ...